docs/buildQueryString.md

```diff
@@ -1,3 +1,7 @@
+<!--meta-description
+Documentation on m.buildQueryString(), which converts an object like {a: "1", b: "2"} into a string like "a=1&b=2"
+-->
+
 # buildQueryString(object)
```

Comment:
`object`is all lowercase here, but `Object` below. Is this correct?

Reply:
@tbreuss Lowercase is fine, as it's consistent with the prose. You can see it in line 26 -> 30 in this same file.